### DISCUSSION TITLE
HELP : I accidentally run chmod 600 /usr

### DISCUSSION THREAD

**Discussion by oddwatcher** [termux/termux-app#2685] [Q&A] [answered]
I had some trouble when installing a self make python package package os reported it cannot access the /usr which is read only so i did try to chmod it and mistyped the 7 to 6 which is chmod 600 /usr. Now i cannot use any command except cd . how can i fix this?

**Comment by tareksander** [comment]
You could try running `/data/data/com.termux/files/usr/bin/chmod 700 /usr`. The `chmod` included in Termux may still work. Also, what is the error you get for other commands?

**Comment by sylirre** [comment, marked as answer]
Start a fail safe shell as per https://wiki.termux.com/wiki/Recover_a_broken_environment and chmod things again to more wide permissions such as `700`.
